instrumentation/opentelemetry-instrumentation-dbapi/src/opentelemetry/instrumentation/dbapi/__init__.py

```diff
@@ -423,7 +423,6 @@ def traced_execution(
         with self._db_api_integration._tracer.start_as_current_span(
             name, kind=SpanKind.CLIENT
         ) as span:
-            self._populate_span(span, cursor, *args)
```

Comment:
Maybe we can add a `span.is_recording`optimization while we are it?

Reply:
Yea!! Added in [195a1b8](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2935/commits/195a1b88640bc746c2790783b7e65aa185cf59fc)